libpod/container_inspect_freebsd.go

```diff
@@ -15,5 +15,14 @@ func (c *Container) platformInspectContainerHostConfig(ctrSpec *spec.Spec, hostC
 	// UTS namespace mode
 	hostConfig.UTSMode = c.NamespaceMode(spec.UTSNamespace, ctrSpec)
 
+	// Devices
+	// Do not include if privileged - assumed that all devices will be
+	// included.
+	var err error
+	hostConfig.Devices, err = c.GetDevices(hostConfig.Privileged, *ctrSpec, map[string]string{})
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	// Do not include devices if privileged - assumed that all devices will be included.
	hostConfig.Devices, err := c.GetDevices(hostConfig.Privileged, *ctrSpec, map[string]string{})
	if err != nil {
		return err
	}
```

Reply:
Nit

Reply:
it doesn't work because it assigns to a struct field not a variable

Reply:
Exactly - I didn't want the declaration of err but nothing else in this block has created the variable, The corresponding code in container_inspect_linux.go also declares err for the same reason.